src/app/hearing-part/components/listing-create/listing-create.component.ts

```diff
@@ -267,6 +265,17 @@ export class ListingCreateComponent implements OnInit {
         });
     }
 
+    private getDurationToDisplay() {
+        if (this.listing.hearing.duration) {
```

Comment:
Is this check needed here and also this return undefined?

Reply:
yes, it's needed as later steps either transforms or tries to use a moment method, 
yes intentionally undefined, though in theory could be just this.listing.hearing.duration - but that would result in potentially more than one empty return value

Reply:
I don't understand what you mean at all. Please explain once more.

Reply:
Q: Is this check needed here and also this return undefined?
A: Yes

Reply:
Taken offline, shortly check is needed for initial step and later whenever input for duration changes this method is called by FormControl to show user proper value = calculations of this method or just undefined/empty